### PR TITLE
Update tasks and refine esthetics

### PR DESCRIPTION
**Pull request by arzafran**
Added new grunt tasks and remove deprecated ones, as defined in gruntfile.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,59 @@
 MSUIF - Making Sense's UI Development Framework
 ==============
 
-So, these are all the tasks we automatized in detail:
+### So, these are all the tasks we automatized in detail:
 
+**grunt-webfont**
```
Comment (andresmoschini) [inline README.md]: What do you think about to use titles in place of bold?

for example for `<H1>`:

# grunt-webfont

or for `<H4>`:

#### grunt-webfont

the main advantage is that titles have ids and could be linked.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,59 @@
...
+
+By default, Bower downloads the dependencies on a single folder inside the project. Once Bower downloads all the technical dependencies, it relocates each one in different folders that we have previously specified.
+
+**grunt-contrib-uglify**
+ 
+This task minifies and drastically reduces file sizes by trimming out spaces and comments in the files.
 
-To run the proyect, just execute the command grunt run.
+To run the project, just execute the command: 
+> grunt run
```
Comment (andresmoschini) [inline README.md]: You can use triple tilde to create code (or command line) blocks `

File: README.md (hunk 1 above)
Comment (damianmuti) [inline README.md]: +1 to @andresmoschini

File: README.md [hunk 3]
```diff
@@ -1,56 +1,59 @@
 MSUIF - Making Sense's UI Development Framework
 ==============
 
-So, these are all the tasks we automatized in detail:
+### So, these are all the tasks we automatized in detail:
```
Comment (andresmoschini) [inline README.md]: I am not sure if it _is_ a title.

**Comment by andresmoschini** [comment]
Thanks @arzafran! it is really nice to have some format in the readme, it is much more friendly!

**Pull request merged**